lsp-completion.el

```diff
@@ -501,6 +502,8 @@ Others: TRIGGER-CHARS"
                           (lsp:completion-item-additional-text-edits? resolved-item)))
                cleanup-fn))))
 
+        (when command? (lsp--execute-command command?))
+
         (when (and lsp-signature-auto-activate
                    (lsp-feature? "textDocument/signatureHelp"))
```

Comment:
this is also related to the PR. Should we remove it and let the server ask us to do that?

Reply:
Since we have user options on this, I would say just leave this as it and maybe changing the default to `nil`.
WDYT?

Reply:
Make sense. We should also look for the editor specific command for triggering completion/signature help(I am not in front of my PC) and implement them in lsp-mode. I think that dart server sends such commands.

Reply:
@ericdallo

Reply:
I'm not sure dart has that, [this is the dart server lsp documentation](https://github.com/dart-lang/sdk/blob/master/pkg/analysis_server/tool/lsp_spec/README.md).

Reply:
according to https://github.com/microsoft/language-server-protocol/issues/905 the commands are: editor.action.triggerSuggest and editor.action.triggerParameterHints . I think that in order to handle this we should introduce something like  `lsp--default-notification-handlers` but for actions.  I am not sure if there is something that can trigger those.

Reply:
I would put that in another PR since it will be another scope not covered in this. To make sure not braking user, we can turn `lsp-signature-auto-activate` back to `t` and disable it once the default notification handlers is implemented.
WDYT?

Reply:
I am fine if you are going to work on that after this PR. Otherwise, we risk getting errors on after completion because we will call the server and it will show an error.

Reply:
Also, I am it will be nice if we find a server that actually sends these commands. I am not familiar with that part of the spec so we might need declaring client capability to enable particular editor command.

Reply:
I found this https://github.com/scalameta/metals/pull/552, but we need to disguised as VsCode for it actually works

Reply:
I suggest opening scala metals bug report and eventually testing with rust analyzer - https://github.com/rust-analyzer/rust-analyzer/blob/master/crates/rust-analyzer/src/to_proto.rs#L212

Reply:
Checked it. We should wrap `lsp--execute-command` with demoted error to prevent blocking user

Reply:
Yes, we can have message like "When you see an error here please open an issue in lsp-mode for implementing the method "XX"

Reply:
Done